### PR TITLE
remove pull-core, update to standard style

### PR DESCRIPTION
**Pull request by ahdinosaur**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -1,46 +1,46 @@
...
+  function next () {
+    if (--n) return
     cb(abort)
   }
-  if(!n) next()
+  if (!n) next()
 }
 
-module.exports = pull.Source(function (streams) {
+module.exports = function pullCat (streams) {
```
Comment (dominictarr) [inline index.js:18]: why did you add a name? this isn't refered to again.

**Comment by dominictarr** [comment]
merged into 1.1.10 (but with "standard style" commit reverted, sorry)
had some this was actually out of date (1.1.9 was on npm but not on github) but fixed now.

File: index.js (hunk 1 above)
Comment (ahdinosaur) [inline index.js:18]: i prefer functions to be named for in-browser debugging purposes.
